packages/rfw/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Updates minimum supported SDK version to Flutter 3.10/Dart 3.0.
 * Fixes lint warnings.
+* Add one core widget and one material widget.
```

Comment:
you should say which ones :-)
remember the old trick when writing documentation: what question is someone trying to answer when reading your prose? what questions will they have after reading your prose?